nengo_bones/scripts/check_notice.py

```diff
@@ -29,59 +30,32 @@ def check_notice(root, add_notice=False, verbose=False):
         Number of files missing a notice.
     """
 
-    notice_lines = NOTICE.splitlines(keepends=True)
+    click.echo("Checking for license text in python files:")
+
     checked = 0
     missing = 0
     for path, _, files in os.walk(root):
         for name in files:
             if name.endswith(".py"):
```

Comment:
Should we be doing more than just Python files? NengoLoihi has some C / C++ files. But it's true that NengoBones is really only set up to work with Python repos so probably this is fine.

Reply:
Yeah I was thinking about that, but a lot of our C/C++ files are actually templated, so this wouldn't work for them anyway. So I thought we might as well just keep things simple here.